core/app/models/spree/taxon.rb

```diff
@@ -59,9 +59,8 @@ def seo_title
     # Sets this taxons permalink to a valid url encoded string based on its
     # name and its parents permalink (if present.)
     def set_permalink
-      permalink_tail = permalink.split('/').last if permalink.present?
-      permalink_tail ||= Spree::Config.taxon_url_parametizer_class.parameterize(name)
-      self.permalink_part = permalink_tail
+      permalink_tail = permalink.present? ? permalink.split('/').last : name
```

Comment:
Do you think we can use [`permalink_part_changed?`](https://api.rubyonrails.org/classes/ActiveModel/Dirty.html) here to change the permalink_part only when it is changed?

Reply:
I'll change that, it's better yes.

Reply:
Actually, we cannot really use that condition because this method is called also for all children via:
```    
after_update :update_child_permalinks, if: :saved_change_to_permalink?
...

def update_permalinks
  set_permalink

  # This will trigger update_child_permalinks if permalink has changed
  save!
end
````
it breaks current test, I will leave it this way.

Reply:
Makes sense, can't see any clean solution here. I agree it's better to keep it as is.